connector/connector_ldap.go

```diff
@@ -154,11 +157,29 @@ func (cfg *LDAPConnectorConfig) Connector(ns url.URL, lf oidc.LoginFunc, tpls *t
 		tlsConfig.Certificates = []tls.Certificate{cert}
 	}
 
+	var initConn, maxConn int
+	if cfg.InitConn != 0 {
+		initConn = cfg.InitConn
+	} else {
+		initConn = 5
+	}
+	if cfg.MaxConn != 0 {
```

Comment:
I have deliberately created a temporary variable here and other places in the constructor to avoid modifying cfg.

My reasoning behind this is:

1. Do not to modify data structures created by parts of the code that I do not control directly.
2. If we at one point want to implement online reconfiguration of dex and all its connectors we probably do need to know the previous state of cfg to have something to compare with values read from disk to be able to determine whether a update is necessary for each value.

If you still do not agree with it I will change it, but at least you know the thought behind it.